diff --git a/colors/gruvbox.vim b/colors/gruvbox.vim
index a0c2c0b..2dfd56f 100644
--- a/colors/gruvbox.vim
+++ b/colors/gruvbox.vim
@@ -1375,8 +1375,17 @@ hi! link jsonBraces GruvboxFg1
 hi! link jsonString GruvboxFg1
 
 " }}}
-
-
+" Ada: {{{
+call s:HL('adaInc', s:aqua, s:none, s:bold)
+call s:HL('adaSpecial', s:aqua, s:none, s:bold)
+hi! link adaKeyword GruvboxPurple
+hi! link adaBegin GruvboxPurple
+hi! link adaEnd GruvboxPurple
+call s:HL('adaTypedef', s:blue, s:none, s:bold)
+call s:HL('adaAssignment', s:aqua, s:none, s:bold)
+hi! link adaAttribute GruvboxGreen
+hi! link adaOperator GruvboxBlue
+" }}}
 " Functions -------------------------------------------------------------------
 " Search Highlighting Cursor {{{
 
