diff --git a/autoload/header.vim b/autoload/header.vim
index d873d56..8baf286 100644
--- a/autoload/header.vim
+++ b/autoload/header.vim
@@ -194,6 +194,14 @@ fun s:set_props()
     elseif b:filetype == 'dosini'
         let b:first_line = '; vim:ft=dosini'
         let b:comment_char = ';'
+    " ----------------------------------
+    elseif b:filetype == 'tcl'
+       let b:first_line = '#!/usr/bin/env tclsh'
+       let b:first_line_pattern = '#!\s*/usr/bin/env\s* tclsh'
+       let b:comment_char = '#'
+    " ----------------------------------
+    elseif b:filetype == 'ada'
+        let b:comment_char = '--'
     " ----------------------------------
 	elseif b:filetype == 'markdown'
 		let b:comment_start = '```'
