diff --git a/autoload/header.vim b/autoload/header.vim
index bb204b3..0170044 100644
--- a/autoload/header.vim
+++ b/autoload/header.vim
@@ -189,6 +192,9 @@ fun s:set_props()
         let b:first_line = '..'
         let b:comment_char = '  '
     " ----------------------------------
+    elseif b:filetype == 'ada'
+        let b:comment_char = '--'
+    " ----------------------------------
     else
         let b:is_filetype_available = 0
     endif
